.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report bugs or errors
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Since `flepiMoP` is mostly a CLI tool/loose collection of software packages maybe it makes more sense to given an example like:
```md
Please provide steps (see below for example) or a code snippet to reproduce.
1. Run this command: '...'
2. Expect this error message: '...'
```